inductiva/tasks/task.py

```diff
@@ -488,9 +488,8 @@ def get_machine_type(self) -> Optional[str]:
             return None
 
         machine_info = info["executer"]
-        if "vm_type" not in machine_info:
-            return "inductiva-machine"
 
+        machine_provider = machine_info["host_type"]
```

Comment:
I've seen `provider`, `provider_id` and here `host_type` referring to the provider. Can't we have a common name?
Do you know why the provider is being returned as `host_type`?

Reply:
FYI @lneves-ai
Can we standardize this nomenclature?